pup-0002.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP documents docstring style conventions for Pulp 3.0.
+
+## Motivation
+
+To improve docstring readability and increase information density.
+
+## Details
+
+Upon acceptance, these changes will be documented in the Pulp style guide.
+
+### Style Choices
+
+Docstrings should generally follow the conventions provided in the comments section of the
```

Comment:
`s/generally //`?

For concision, Docstrings should follow the conventions as documented, except where Pulp-specific additions or exceptions apply.